kong.conf.default

```diff
@@ -1618,6 +1618,15 @@
 # TUNING & BEHAVIOR
 #------------------------------------------------------------------------------
 
+#privileged_agent = off          # Enables or disables privileged agent worker
```

Comment:
This makes sense. Customers only need to turn it on when having CP/DP sync issues.

It is a trade-off between memory and performance. Compared to the total memory Kong requires, this might be acceptable to cusotmers.